airflow/providers/common/sql/hooks/sql.py

```diff
@@ -50,6 +50,7 @@
     from airflow.providers.openlineage.sqlparser import DatabaseInfo
 
 T = TypeVar("T")
+DEFAULT_SQL_PLACEHOLDERS = frozenset({"%s", "?"})
```

Comment:
```suggestion
SQL_PLACEHOLDERS = frozenset({"%s", "?"})
```

It's not defaults it's allowed placeholders